references/sdk.md

```diff
@@ -69,639 +69,503 @@ Returns:
 ```python
-Client.link_dataset_with_objective(self, dataset_key, objective_key)
+Client.link_dataset_with_objective(self, dataset_key: str, objective_key: str) -> str
 ```
 Link dataset with objective.
 ## link_dataset_with_data_samples
 ```python
-Client.link_dataset_with_data_samples(self, dataset_key, data_sample_keys)
+Client.link_dataset_with_data_samples(self, dataset_key: str, data_sample_keys: str) -> List[str]
 ```
 Link dataset with data samples.
 ## download_dataset
 ```python
-Client.download_dataset(self, key, destination_folder)
+Client.download_dataset(self, key: str, destination_folder: str) -> None
 ```
 Download data manager resource.
 
 Download opener script in destination folder.
 
 ## download_algo
 ```python
-Client.download_algo(self, key, destination_folder)
+Client.download_algo(self, key: str, destination_folder: str) -> None
 ```
 Download algo resource.
 
 Download algo package in destination folder.
 
 ## download_aggregate_algo
 ```python
-Client.download_aggregate_algo(self, key, destination_folder)
+Client.download_aggregate_algo(self, key: str, destination_folder: str) -> None
 ```
 Download aggregate algo resource.
 
 Download aggregate algo package in destination folder.
 
 ## download_composite_algo
 ```python
-Client.download_composite_algo(self, key, destination_folder)
+Client.download_composite_algo(self, key: str, destination_folder: str) -> None
 ```
 Download composite algo resource.
 
 Download composite algo package in destination folder.
 
 ## download_objective
 ```python
-Client.download_objective(self, key, destination_folder)
+Client.download_objective(self, key: str, destination_folder: str) -> None
 ```
 Download objective resource.
 
 Download metrics script in destination folder.
 
 ## describe_algo
 ```python
-Client.describe_algo(self, key)
+Client.describe_algo(self, key: str) -> str
 ```
 Get algo description.
 ## describe_aggregate_algo
 ```python
-Client.describe_aggregate_algo(self, key)
+Client.describe_aggregate_algo(self, key: str) -> str
 ```
 Get aggregate algo description.
 ## describe_composite_algo
 ```python
-Client.describe_composite_algo(self, key)
+Client.describe_composite_algo(self, key: str) -> str
 ```
 Get composite algo description.
 ## describe_dataset
 ```python
-Client.describe_dataset(self, key)
+Client.describe_dataset(self, key: str) -> str
 ```
 Get dataset description.
 ## describe_objective
 ```python
-Client.describe_objective(self, key)
+Client.describe_objective(self, key: str) -> str
 ```
 Get objective description.
 ## leaderboard
 ```python
-Client.leaderboard(self, objective_key, sort='desc')
+Client.leaderboard(self, objective_key: str, sort: str = 'desc') -> str
 ```
 Get objective leaderboard
 ## cancel_compute_plan
 ```python
-Client.cancel_compute_plan(self, compute_plan_id)
+Client.cancel_compute_plan(self, compute_plan_id: str) -> substra.sdk.models.ComputePlan
 ```
 Cancel execution of compute plan.
 # retry_on_exception
```

Comment:
Hi, this one should be an h2 tag, no?
```suggestion
## retry_on_exception
```

Reply:
Hello, 

`retry_on_exception` is actually its own function, not part of the `Client` class, that's why it got an h1 tag.

Reply:
Ah okay I see, thank you!